doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,94 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes in detail the detection of dropping foreign keys in MySQL databases, 
+using Atlas's linting capabilities.
+
+A foreign key constraint can be dropped from a table using the following command:
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_id;
+```
+
+Even though this action does not cause data loss and there may be good reasons to perform it on a production database, it can have negative and sometimes unpredictable side effects which should be taken into consideration.
+Atlas's linter can be used to raise a flag when a foreign key drop is about to be executed. Simply run:
+```bash
+atlas migrate lint --dir file://migrations --dev-url "mysql://root:pass@localhost:3306/dev" --latest 1
+```
+The `--latest` flag tells Atlas to analyze only the last migration file. If the file contains a foreign key drop, the output will be something like:
+
+```bash
+20230305125128.sql: constraint deletion detected:
+
+	L1: Dropping foreign-key constraint "owner_id"
+```
+
+[Click here](/versioned/lint) for more information about how to use the Atlas Linter.
+
+## Impact of dropping foreign key constraints
+
+### Data integrity
+Like mentioned above, a foreign key drop is not a destructive action, meaning it's not causing data loss. But it does impact the **integrity** of the data. In other words, this action won't change the data in the table, but it will change the way different tables interact with one another.
+
+Suppose we have the following database:
+<p style={{textAlign: "center"}}><a href="https://gh.atlasgo.cloud/explore/1b14810d"><img src="https://atlasgo.io/uploads/users-pets.png" alt="Pets ERD"/></a></p>
+
+ And we drop the foreign key between `pets` and `users` with the following command: 
+
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_fk;
+```
+
+Now, inserting data into the `pets` table is still possible, however it is also possible to insert "invalid" data. For example, assuming the `users` table is empty, the following command will not fail:
+
+```sql
+INSERT INTO pets (name, owner_id) VALUES ('Fortuna', 1);
+```
```

Comment:
```suggestion
```

Reply:
@rotemtam don't you think it's nice to have? if we give a full example readers can follow along to see the JOIN query fails for themselves.

Reply:
i suggested to move this point above (with slight variation), you accepted that change